web/src/Sidebar.jsx

```diff
@@ -64,10 +48,55 @@ export default function Sidebar() {
   );
 }
 
+function SortedCameras({ unsortedCameras }) {
+
+  const sortedCameras = useMemo(() =>
+    Object.entries(unsortedCameras)
```

Comment:
By calculating the sortedCameras here, you end up running it for both `SortedCameras` and `SortedRecordingCameras`. Can you move this into the parent and pass it to the functions instead?

Reply:
Thank you, should be good now. Note that I had to have the internal check on the config because if I put `useMemo` after the main (!config) check it complains of being conditional. This seems to work well (yay efficiency) 😄